app/views/staff/pages/_page.html.haml

```diff
@@ -1,3 +1,7 @@
 %tr
   %td= link_to(page.name, edit_event_staff_page_path(current_event, page))
   %td= link_to(page.slug, edit_event_staff_page_path(current_event, page))
+  %td= button_to("Publish",
+    publish_event_staff_page_path(current_event, page),
+    data: { confirm: "Are you sure you want to do publish?" },
```

Comment:
Typo, "do publish"

Reply:
I saw this as well and have it fixed in the next PR.